src/resty/gcp/init.lua

```diff
@@ -5,7 +5,7 @@ local http = require "resty.luasocket.http"
 local lookup_helper = function(self, key) -- signature to match __index meta-method
     if type(key) == "string" then
         local lckey = key:lower()
-        for k, v in pairs(self) do
+        for k, _ in pairs(self) do
```

Comment:
Maybe we could add linting to the CI?

Reply:
Sure, but let's do that separately.
https://github.com/Kong/lua-resty-gcp/issues/20